src/rard/research/models/fragment.py

```diff
@@ -212,6 +212,11 @@ def related_queryset(self):
     # the fragments that we are apposita for...
     fragments = models.ManyToManyField("Fragment", blank=True, related_name="apposita")
 
+    # the anonymous fragments that we are apposita for
+    anonymous_fragments = models.ManyToManyField(
+        "self", blank=True, symmetrical=False, related_name="anonymous_apposita"
```

Comment:
Why would it not be symmetrical? If you add AF2 as an apposita to AF1 is AF1 not also an apposita of AF2?

Reply:
Yeah, I asked Richard about that and he said it shouldn't be symmetrical. See #374